packages/lodestar/src/chain/chain.ts

```diff
@@ -253,6 +255,27 @@ export class BeaconChain implements IBeaconChain {
     this.logger.info("Beacon chain initialized");
   }
 
+  /**
+   * Initialize our chain from a weak subjectivity state.
+   * @param state a weak subjectivity state
+   */
+  public async initializeWeakSubjectivityState(weakSubjectivityState: TreeBacked<BeaconState>): Promise<void> {
```

Comment:
Can we just reuse `initializeChain` and `initForkChoice`? These methods should be doing what we need already (should be tweaked if need be).